testsuite/tests/kuadrant/reconciliation/cr_configuration/test_reconcile_limitador.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for Kuadrant sub component - Limitador CR configuration via Kuadrant CR"""
+
+import pytest
+
+pytestmark = [pytest.mark.kuadrant_only]
```

Comment:
This is a disruptive test and we should probably mark it as such (possibly in the future but very soon)